mipengine/node/tasks/merge_tables.py

```diff
@@ -19,33 +20,31 @@ def get_merge_tables(context_id: str) -> str:
 
         Returns
         ------
-        str(List[str])
-            A list of merged table names in a jsonified format
+        List[str]
+            A list of merge table names
     """
-    return json.dumps(merge_tables.get_merge_tables_names(context_id))
+    return merge_tables.get_merge_tables_names(context_id)
 
 
-# TODO Add in method description the jsonified input types
 @shared_task
-def create_merge_table(context_id: str, command_id: str, partition_table_names_json: str) -> str:
+def create_merge_table(context_id: str, command_id: str, partition_table_names: List[str]) -> str:
     """
         Parameters
         ----------
         context_id : str
             The id of the experiment
         command_id : str
             The id of the command that the merge table
-        partition_table_names_json : str(List[str])
-            Its a list of names of the tables to be merged in a jsonified format
+        partition_table_names: List[str]
```

Comment:
Do we really need to add here a new concept, partition tables? A merge table is just a merge of some tables, can't the input parameter just be named table_names?